sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/CSVExprUtils.scala

```diff
@@ -120,6 +120,11 @@ object CSVExprUtils {
    * @throws SparkIllegalArgumentException if any of the individual input chunks are illegal
    */
   def toDelimiterStr(str: String): String = {
+    if (str == null) {
```

Comment:
QQ: Is "" valid? Do we know how users specified null?

Reply:
spark.read.format("csv")
.option("delimiter", null)
.load()

for example, also at line 71 error is thrown for ""